modules/telemetry/packet_spec/blocks.py

```diff
@@ -225,6 +226,27 @@ def output_formatted(self, into: dict[str, Any]):
         add_to_dict(into, ["magnetic_field", "magnitude"], magnitude(self.x_axis, self.y_axis, self.z_axis))
 
 
+@dataclass
+class FlightStatus(TimedBlock):
+    _struct_format: str = field(default="<hB", init=False, repr=False)
+    flight_status: int 
+    def output_formatted(self, into: dict[str, Any]):
+        add_to_dict(into, ["flight_status", "flight_status"], self.config.parsing_parameters["status_codes"][self.flight_status])
```

Comment:
Don't forget to add the mission time to the dict as well, the identifier for the status itself can just be "status"